source/includes/_suggestions.md

```diff
@@ -65,4 +68,5 @@ content            | The suggestion content
 feeling            | Feeling of the suggestion can be ("sad", "unhappy", "ok", "satisfied", "happy")
 custom_tag_id      | ID of the custom tag for the suggestion
 not_anonymous      | Boolean value. Displays if suggestion is anonymous or not.
+terminal_spot_id | optional | ID of the terminal spot for the suggestion.
```

Comment:
What happens if the new row is the only one with 3 columns?

Reply:
Good catch. I removed the optional column.